src/main/webapp/view/discussion/discussion.js

```diff
@@ -39,6 +39,8 @@ const SELECTOR_REPLY_TEXTAREA = '#reply-textarea';
 // 10 seconds.
 const TIME_TOLERANCE_MS = 10000;
 
+window.postNewComment = postNewComment;
```

Comment:
Should we be able to eliminate window. here?

Reply:
Unfortunately not at the moment, because it is used in the HTML